linera-service/src/cli_wrappers/kind_wrapper.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use tokio::process::Command;
+
+use crate::util::CommandExt;
+
+pub struct KindWrapper;
```

Comment:
(see Docker comment on this) This one the state could make sense just because `cluster_id` is a common parameter in all static method calls, so I can do it